src/main/java/io/github/thebusybiscuit/slimefun4/implementation/items/multiblocks/EnhancedCraftingTable.java

```diff
@@ -53,7 +53,7 @@ public void onInteract(Player p, Block b) {
                 }
             }
 
-            if (inv.isEmpty()) {
+            if (SlimefunUtils.isInventoryEmpty(inv)) {
```

Comment:
There is more than one machine in Slimefun, this only fixes it for the Enhanced Crafting Table.
At least two more cases just by text search alone...

https://github.com/Slimefun/Slimefun4/blob/8c0ef8a97fbb289310c3f9aacf26ca6c76be8867/src/main/java/io/github/thebusybiscuit/slimefun4/implementation/items/multiblocks/MagicWorkbench.java#L63

https://github.com/Slimefun/Slimefun4/blob/8c0ef8a97fbb289310c3f9aacf26ca6c76be8867/src/main/java/io/github/thebusybiscuit/slimefun4/implementation/items/multiblocks/ArmorForge.java#L54

Come on, guys...

@svr333 @Sfiguz7 Did you actually review this or just blindly approve because Walshy told you to? :eyes:

Reply:
I reviewed the code that changed on this one I didn't check for other places where this might need an implementation  
my bad